### ISSUE TITLE
`programs/tokens.chef` raises parse error

### ISSUE THREAD

**Issue by eggplants** [booleancoercion/rchef#2]
```shellconsole
$ cargo run programs/tokens.chef
warning: use of deprecated associated function `std::array::IntoIter::<T, N>::new`: use `IntoIterator::into_iter` instead
   --> src/parser.rs:655:31
    |
655 |         let kinds = IntoIter::new(kinds);
    |                               ^^^
    |
    = note: `#[warn(deprecated)]` on by default

warning: `rchef` (lib) generated 1 warning
    Finished dev [unoptimized + debuginfo] target(s) in 0.02s
     Running `target/debug/rchef programs/tokens.chef`
[line 28] syntax error: expected FULLSTOP, found INTO
[line 29] syntax error: expected FULLSTOP, found INTO
[line 59] syntax error: expected IDENTIFIER, found SERVES
[line 59] syntax error: expected IDENTIFIER, found NUMBER
[line 59] syntax error: expected IDENTIFIER, found FULLSTOP
parse error
```

It seems that `Add` method with `INTO` is not valid for rchef parser like line 28 or 29.

After replacing `into` with `to`, the parser raises `[line 71] syntax error: expected IDENTIFIER, found NEWLINE`.

**Comment by JanEricNitschke** [comment]
You need to change `into` to `to` as you already discovered. However you also cant end your file with a newline. This is  the second error you see and not present in the version on github.

**Comment by eggplants** [comment]
[`programs/tokens.chef`](https://github.com/booleancoercion/rchef/blob/main/programs/tokens.chef) is not mine but this repo's. Anyway, one of original author's sample codes ends with a newline.
- https://dangermouse.net/esoteric/chef_hello.html (ends with `<br>`)
- https://dangermouse.net/esoteric/chef_fib.html (ends without `<br>`)

So the rchef interpreter should accept end-of-file line break(s). Otherwise, it should be documented that they are unacceptable.

**Comment by JanEricNitschke** [comment]
Yes, i know. I just wanted to let you know that. I forked this repo, corrected the `tokens.chef` file there and adjusted it to handle trailing newlines (although in a pretty hacky way).

However the sample texts on the DM website are also not actually conforment to the spec as described on the website and the fib program is also broken. So i would cut the original author here some slack.
